tests/testthat/test_featsel_filters.R

```diff
@@ -109,7 +109,8 @@ test_that("randomForestSRC_var.select minimal depth filter returns NA for featur
     method = "randomForestSRC_var.select",
     nselect = 5,
     more.args = list("randomForestSRC_var.select" = list(method = "md", nrep = 5)))
-  expect_equal(is.na(dat$data$value[dat$data$name %in% c("Sepal.Length", "Sepal.width")]), TRUE)
+  expect_equal(all(is.na(dat$data$value[dat$data$name %in% c("Sepal.Length", "Sepal.Width")])), TRUE)
+  expect_equal(all(is.na(dat$data$value)), FALSE)
```

Comment:
Could you check explicitly for the ones that should not be NA please?

Reply:
OK. Done.